core/src/main/resources/hudson/model/Label/Badge/index.jelly

```diff
@@ -0,0 +1,30 @@
+<!--
+The MIT License
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" >
+  <st:adjunct includes="io.jenkins.plugins.bootstrap5"/>
+  
+  <j:set var="cssSuffix" value="${it.name.replace(':', '-').replace('.', '-')}"/>
```

Comment:
This sanitization expression is probably too simple: Label atoms can contain spaces, non-ASCII chars, etc.

Reply:
1. Configure a (freestyle) job to reference the label `"lol wut"`.
2. Go to `/label/lol%20wut/` and see a label page.

Reply:
it shall works now anywhere, because my id is now the color self
![image](https://user-images.githubusercontent.com/89339813/178424527-3e7445c2-ac85-4466-976a-fb2b17a5a274.png)